temporalcli/commands.workflow_view.go

```diff
@@ -102,36 +102,47 @@ func (c *TemporalWorkflowDescribeCommand) run(cctx *CommandContext, args []strin
 
 	cctx.Printer.Println(color.MagentaString("Execution Info:"))
 	info := resp.WorkflowExecutionInfo
+	extendedInfo := resp.WorkflowExtendedInfo
 	_ = cctx.Printer.PrintStructured(struct {
-		WorkflowId           string
-		RunId                string
-		Type                 string
-		Namespace            string
-		TaskQueue            string
-		AssignedBuildId      string
-		StartTime            time.Time
-		CloseTime            time.Time                  `cli:",cardOmitEmpty"`
-		ExecutionTime        time.Time                  `cli:",cardOmitEmpty"`
-		Memo                 map[string]*common.Payload `cli:",cardOmitEmpty"`
-		SearchAttributes     map[string]*common.Payload `cli:",cardOmitEmpty"`
-		StateTransitionCount int64
-		HistoryLength        int64
-		HistorySize          int64
+		WorkflowId              string
+		RunId                   string
+		Type                    string
+		Namespace               string
+		TaskQueue               string
+		AssignedBuildId         string
+		StartTime               time.Time
+		CloseTime               time.Time                  `cli:",cardOmitEmpty"`
+		ExecutionTime           time.Time                  `cli:",cardOmitEmpty"`
+		Memo                    map[string]*common.Payload `cli:",cardOmitEmpty"`
+		SearchAttributes        map[string]*common.Payload `cli:",cardOmitEmpty"`
+		StateTransitionCount    int64
+		HistoryLength           int64
+		HistorySize             int64
+		ExecutionExpirationTime time.Time
+		RunExpirationTime       time.Time
+		CancelRequested         bool
+		LastResetTime           time.Time
+		OriginalStartTime       time.Time
```

Comment:
I think we should add the `cli:",cardOmitEmpty"` on any of these that can have zero values, but that can be done in a subsequent PR if there are time constraints since there are no compatibility expectations with text output